lib/adapter.js

```diff
@@ -113,21 +111,25 @@ module.exports = (function main() {
     },
 
     teardown(callback) {
-      if (this._dbConnected) {
+      if (dbConnected) {
         db.close(callback);
       } else {
         callback();
       }
     },
 
-
-    query(collectionName, statement, data, callback) {
-      if (_.isFunction(data)) {
-        callback = data;
-        data = null;
+    teardownSync() {
+      if (dbConnected) {
+        try {
+          db.closeSync();
+// eslint-disable-next-line no-empty
```

Comment:
it's bad idea to mute it (http://stackoverflow.com/questions/1234343/why-are-empty-catch-blocks-a-bad-idea)